geps/gep-1897/index.md

```diff
@@ -238,35 +240,23 @@ Thus, the following additions would be made to the Gateway API:
-| TLSRoute   | Listener Mode: Passthrough | Yes                                           | No                            |
-| TLSRoute   | Listener Mode: Terminate   | Yes                                           | Implementation-dependent      |
-| TLSRoute   | Listener Mode: Passthrough | No                                            | No                            |
-| TLSRoute   | Listener Mode: Terminate   | No                                            | No                            |
-| TCPRoute   | Listener TLS               | Yes                                           | Implementation-dependent      |
-| TCPRoute   | No listener TLS            | Yes                                           | Implementation-dependent      |
-| TCPRoute   | Listener TLS               | No                                            | No                            |
-| TCPRoute   | No listener TLS            | No                                            | No                            |
-| UDPRoute   | Listener TLS               | Yes                                           | No                            |
-| UDPRoute   | No listener TLS            | Yes                                           | No                            |
-| UDPRoute   | Listener TLS               | No                                            | No                            |
-| UDPRoute   | No listener TLS            | No                                            | No                            |
-| GRPCRoute  | Listener TLS               | Yes                                           | Implementation-dependent      |
-| GRPCRoute  | No Listener TLS            | Yes                                           | Implementation-dependent      |
-| GRPCRoute  | Listener TLS               | No                                            | No                            |
-| GRPCRoute  | No Listener TLS            | No                                            | No                            |
+The `BackendTLSPolicy` tells a client "Connect to this service using TLS".
+This is unconditional to the type of traffic the gateway client is forwarding.
+For instance, the following will all have the gateway client add TLS:
+
+* A Gateway accepts traffic on an HTTP listener
+* A Gateway accepts and terminates TLS on an HTTPS listener
+* A Gateway accepts traffic on a TCP listener
+* A Gateway accepts traffic with `Mode: Passthrough` on a TLS listener
+* A service mesh client accepts HTTP traffic
+* A service mesh client accepts TLS traffic
+* A service mesh client accepts TCP traffic
+
+Note that there are cases where these patterns may result in multiple layers of TLS on a single connection.
+There may be even cases where the gateway implementation is unaware of this; for example, processing TCPRoute traffic -- the traffic may or may not be TLS, and the gateway would be unaware.
```

Comment:
TCPRoute is excluded from this GEP.

Reply:
This PR is removing the exclusion

Reply:
TCPRoute and GRPCRotue are excluded in the section marked "Longer Term Goal", with the preamble: "These are worthy goals, but deserve a different GEP for proper attention."

Reply:
To make the semantic change that "BackendTLSPolicy means to send TLS to the service" (vs today where its conditional based on route type), this inherently applies to route types like TCPRoute and GRPCRoute as well. So this expansion of the GEP seems required.

Its not so much that this GEP now includes TCPRoute/GRPCRoute/etc, its that this GEP now no longer is dependent on routes at all.